README.md

```diff
@@ -0,0 +1,124 @@
+# web3j Sample Project
+
+This project provides a very simple starter application for working with web3j and the test 
+Ethereum network (testnet) known as Rinkeby.
+
+The demo deploys a *hello world* smart contract which contains a greeting message. The current 
+greeting message can be read and modified in the smart contract.  
+
+In order to run this demo, three pieces of information are required:
+
+1. Ethereum client (or node) endpoint. The simplest thing to do is 
+[request a free access token](https://infura.io/register.html) from Infura.
+2. A wallet file. This can be generated using the web3j 
+[command line tools](https://docs.web3j.io/command_line.html) by running the command:
+3.  Some Ether. This can be requested from the [Rinkeby Faucet](https://www.rinkeby.io/#faucet).
+
+- To create a wallet file, run the command:
+```aidl
+web3j wallet create
+```
+
+- You will be asked to secure your wallet with a password. You will have the option
+of choosing a destination directory for your wallet file.
+In the `Application.java` class under the package `org.web3j.sample`,
+on line `66-67` replace the placeholders with your wallet file password and directory path.
+
+- With the access token retrieved from Infura, on line `58` replace the placeholder 
+in the url with your token. You can find this under the `Endpoint` section:
+
+![Alt text](InfuraTokenPointer.png)
```

Comment:
Please get Krystian to do this - it needs to look more professional - or remove it entirely for now.

Reply:
Requested.

Reply:
Please have a look at the new image uploaded.